modules/framework/galasa-parent/dev.galasa.framework.api.cps/build.gradle

```diff
@@ -5,14 +5,16 @@ plugins {
 
 description = 'Galasa API - CPS'
 
-version = '0.37.0'
+version = '0.38.0'
 
 dependencies {
     implementation project(':dev.galasa.framework')
     implementation project(':dev.galasa.framework.api.beans')
     implementation project(':dev.galasa.framework.api.common')
     testImplementation(testFixtures(project(':dev.galasa.framework.api.common')))
-    testImplementation 'org.assertj:assertj-core:3.23.1'
+    testImplementation platform('dev.galasa:dev.galasa.platform:0.38.0')
+
+    testImplementation 'org.assertj:assertj-core:3.23.1' // Overriding version 3.16.1 suggested by the Platform.
```

Comment:
why are we overriding the one suggested by the platform ?

Reply:
Seems we need a story to uplift to the 3.23.1 version, to chase-up this loose end.